core/src/main/resources/schema/h2/h2-create-schema-legacy.sql

```diff
@@ -18,7 +18,7 @@ CREATE TABLE IF NOT EXISTS PUBLIC."snapshot" (
 );
 
 
-CREATE TABLE IF NOT EXISTS "durable_state" (
+CREATE TABLE IF NOT EXISTS PUBLIC."durable_state" (
```

Comment:
It looks like you can set the schema name for an h2 connection in the JDBC URL.  I wonder if that could be used to select a different schema before this change, but would break now that `PUBLIC.` is explicit here?

Reply:
the existing sql script has some statements with the "PUBLIC" schema and some without. This just standardises on having the schema explicit in all statements.